units/include/scipp/units/dummy.h

```diff
@@ -25,8 +25,9 @@ SCIPP_UNITS_DECLARE_DIMENSIONS(X, Y, Z)
 
 template <> struct supported_units<dummy::Unit> {
   using type = decltype(detail::make_unit(
-      std::make_tuple(m), std::make_tuple(dimensionless, dimensionless / m, s,
-                                          dimensionless / s, m / s)));
+      std::make_tuple(m),
+      std::make_tuple(dimensionless, rad, deg, dimensionless / m, s,
```

Comment:
So you don't need `rad / deg` and `deg / rad` for the dummy set?

Reply:
Likely `core` will actually fail to compile if we move away from the default `neutron` units. I fear we have probably many issues that have crept into the tests, requiring special units that are not part of `dummy`... I am not sure how much of a problem this is. The "right" thing would be to change the CI to also compile with units other than `neutron`, ensuring that we stay consistent.

I would have liked to avoid adding `rad` and `deg` but that would have required other weird code like we did for `counts_unit`.